scipy/_lib/src/ccallback.h

```diff
@@ -265,6 +265,62 @@ static void ccallback__err_invalid_signature(ccallback_signature_t *signatures,
     return;
 }
 
+/*
+ * Retrieve the LowLevelCallable Type
+ *
+ * Parameters
+ * ----------
+ * callback_obj : PyObject
+ *     Object provided by the user. Usually, LowLevelCallback object, or a
+ *     Python callable.
+ *
+ * Returns
+ * -------
+ * lowlevelcallable_type : PyTypeObject *
+ *     !=NULL if success, ==NULL on failure.
+ *
+ */
+ static PyTypeObject * get_lowlevelcallable_type() {
```

Comment:
Function names should obey the naming convention in the file, for public and private functions.